### PR TITLE
Green Table Additions

### PR DESCRIPTION
**Pull request by puja-trivedi**

### CODE REVIEW

**Comment by satra** [comment]
shouldn't this only add the yaml file? the rest should be auto generated on merge.

**Comment by puja-trivedi** [comment]
@satra - if you look at the workflow it is generated on "push" as well as "pull request". Although it might make more sense to change it to "merge" and "push. 

`name: generating other formats

on:
  push:
  pull_request:

`

**Comment by satra** [comment]
sorry i had forgotten. the conflicts misled me. yes, it would be good to test that the autogeneration works, but the actual autogeneration could happen when ready to merge, or when a label is set on the PR.

**Comment by djarecka** [comment]
@puja-trivedi - perhaps you didn't pull the changes to autogenerated formats and that's why you have the conflicts?

File: linkml-schema/kbmodel.yaml [hunk 1]
```diff
@@ -299,27 +502,69 @@ classes:
...
     slot_usage:
-      wasGeneratedBy:
+      name:
+        description: >- 
+          Name of collection of enriched cells or nuclei after enrichment process (usually via FACS using the Enrichment Plan) applied to dissociated_cell_sample.
+        required: true
+      source barcode name:
+        description: >- 
+          Name of molecular barcode used to individual Enriched Cell Source to allow for pooling of Enriched Cell Sources before 10x load (Barcoding Cell step) [aka 1st round barcodes].
+          wasGeneratedBy:
```
Comment (pbishwakarma) [inline linkml-schema/kbmodel.yaml:563]: Think there's an indentation problem here

File: linkml-schema/kbmodel.yaml [hunk 2]
```diff
@@ -299,27 +502,69 @@ classes:
...
+      - bican
+      - library_generation
+    slots:
+      - source barcode name
+    attributes:
+      histone modification marker:
+        description: >- 
+          Histone modification marker antibodies (eg H3K27ac, H3K27me3, H3K9me3) used in conjunction with an Enriched Cell Source Barcode in order to combine multiple Enriched Cell Populations before Barcoded Cell Sample step for 10xMultiome method. 
+          Each of the Histone antibodies captures an essential part of the epigenome.
+      population:
```
Comment (pbishwakarma) [inline linkml-schema/kbmodel.yaml:550]: Should this be `enrichment population`?

File: linkml-schema/kbmodel.yaml (hunk 1 above)
Comment (satra) [reply linkml-schema/kbmodel.yaml:563]: @djarecka - isn't linting part of the action? if not we should.  i would think the linter would have caught this.

**Comment by satra** [comment]
@puja-trivedi - is this the latest version of the green sheets/minimal metadata release? (and could you please fix the conflicts?)

File: linkml-schema/kbmodel.yaml [hunk 3]
```diff
@@ -438,21 +776,127 @@ classes:
...
+          Local Pool Name is used to communicate sequencing status between SeqCore and Library Labs.
+        required: true
+      xref: 
+        description: >-
+          Library Pool Tube local name. Label of the tube containing the library pool, which is made up of multiple library_aliquots. 
+          This is a Library Lab local tube name, before the pool is aliquoted to the Seq Core provided tube 'Library Pool Tube Name'.
+        required: true
+      quantity fmol:
+        description: Amount of library pool in the tube as measured in femtamoles (fmol)
+      quantity pM:
```
Comment (pbishwakarma) [inline linkml-schema/kbmodel.yaml:838]: Should this be concentration instead of quantity?

File: linkml-schema/kbmodel.yaml [hunk 4]
```diff
@@ -438,21 +776,127 @@ classes:
...
+        description: Sequencer Loading Concentration as measured in pM (pmol/L). This is a value used by the SeqCore.
+      concentration nm:
```
Comment (pbishwakarma) [inline linkml-schema/kbmodel.yaml:840]: `nM` instead of `nm`

File: linkml-schema/kbmodel.yaml (hunk 1 above)
Comment (puja-trivedi) [reply linkml-schema/kbmodel.yaml:563]: @satra - linting is part of the action and it is passing with no errors so I think the indentation is fine.

**Comment by puja-trivedi** [comment]
> @puja-trivedi - is this the latest version of the green sheets/minimal metadata release? (and could you please fix the conflicts?)

Is there a specific data associated with the latest version of the green table you are thinking of? I haven't looked at the green table in a few weeks (ever since I created the PR). I am waiting to update the model until Patrick merges his PR since I have been told that the tables on that repo are the "truth" and I should use those from now on.

**Comment by satra** [comment]
> I am waiting to update the model further until Patrick merges his PR

no you should use patrick's PR, but don't wait for it to be merged. just use that info to update this. note the commit that you are working off of, and any changes relative to that commit can be incorporated later.

**Comment by puja-trivedi** [comment]
@satra - In last week's biolink meeting, Pam suggested that we could use [Biolink:Procedure](https://biolink.github.io/biolink-model/docs/Procedure.html) for the orange boxes and [Biolink:MaterialSample](https://biolink.github.io/biolink-model/docs/MaterialSample.html) for the purple boxes. Would you be able to take a look at these classes? I wanted to run it by you before I update the model.

**Comment by puja-trivedi** [comment]
the checks are currently failing due to a [linkml issue](https://github.com/biolink/biolink-model/issues/1420).

**Pull request closed without merging**